pymc3_hmm/utils.py

```diff
@@ -323,10 +323,16 @@ def plot_fn(ax, data, **kwargs):
 
         plot_fn(ax, split_data, drawstyle=drawstyle, linewidth=linewidth, **plot_kwds)
 
-        ax.xaxis.set_minor_locator(mdates.HourLocator(byhour=range(0, 23, 3)))
-        ax.xaxis.set_minor_formatter(mdates.DateFormatter("%H"))
-        ax.xaxis.set_major_locator(mdates.WeekdayLocator(byweekday=range(0, 7, 1)))
-        ax.xaxis.set_major_formatter(mdates.DateFormatter("%Y-%m-%d %a"))
+        if pd.to_timedelta(f'1{split_freq}')/300 <pd.to_timedelta('1H'):
```

Comment:
This needs some sort of check, in case there is already a number specified in the `split_freq` (e.g. `split_freq = "10d"`).